test/powershell/Modules/Microsoft.PowerShell.Security/GetCredential.Tests.ps1

```diff
@@ -21,11 +21,41 @@ Describe "Get-Credential Test" -tag "CI" {
         $rs.Dispose()
         $ps.Dispose()
     }
-    It "Get-Credential produces as credential object" {
-        $cred = $ps.AddScript("Get-Credential -User Joe -Message Foo").Invoke() | Select-Object -First 1
+    It "Get-Credential with message, produces a credential object" {
+        $cred = $ps.AddScript("Get-Credential -UserName Joe -Message Foo").Invoke() | Select-Object -First 1
         $cred.gettype().FullName | Should Be "System.Management.Automation.PSCredential"
         $netcred = $cred.GetNetworkCredential()
         $netcred.UserName | Should be "Joe"
         $netcred.Password | Should be "this is a test"
+        # Resx file only include english language, this should be changed to reflect Current Culture if title change by language.
+        $th.ui.Streams.Prompt[-1] | should be "Credential:Windows PowerShell credential request.:Foo" 
+    }

```

Comment:
We do run tests on localized builds of Windows, so this test would fail on those builds.

In some tests, if the message is important, we'll actually load the resource to get the text - that works on localized builds as well. We might not have any examples of that pattern in out pester tests though.